examples/vision/yolov8.py

```diff
@@ -0,0 +1,571 @@
+"""
+
+"""
+Below table compares the performance metrics of five different YOLOv8 models with
+different sizes (measured in pixels): YOLOv8n, YOLOv8s, YOLOv8m, YOLOv8l, and YOLOv8x.
+The metrics include mean average precision (mAP) values at different
+intersection-over-union (IoU) thresholds for validation data, inference speed on CPU with
+ONNX format and A100 TensorRT, number of parameters, and number of floating-point
+operations (FLOPs) (both in millions and billions, respectively). As the size of the
+model increases, the mAP, parameters, and FLOPs generally increase while the speed
+decreases. YOLOv8x has the highest mAP, parameters, and FLOPs but also the slowest
+inference speed, while YOLOv8n has the smallest size, fastest inference speed, and lowest
+mAP, parameters, and FLOPs.
+
+| Model                                                                                |
+size<br><sup>(pixels) | mAP<sup>val<br>50-95 | Speed<br><sup>CPU ONNX<br>(ms) |
+Speed<br><sup>A100 TensorRT<br>(ms) | params<br><sup>(M) | FLOPs<br><sup>(B) |
+| ------------------------------------------------------------------------------------ |
+--------------------- | -------------------- | ------------------------------ |
+----------------------------------- | ------------------ | ----------------- |
+| YOLOv8n | 640                   | 37.3                 | 80.4                          
+| 0.99                                | 3.2                | 8.7               |
+| YOLOv8s | 640                   | 44.9                 | 128.4                         
+| 1.20                                | 11.2               | 28.6              |
+| YOLOv8m | 640                   | 50.2                 | 234.7                         
+| 1.83                                | 25.9               | 78.9              |
+| YOLOv8l | 640                   | 52.9                 | 375.2                         
+| 2.39                                | 43.7               | 165.2             |
+| YOLOv8x | 640                   | 53.9                 | 479.1                         
+| 3.53                                | 68.2               | 257.8             |
+"""
+
+"""
+You can read more about YOLOV8 and its architecture in this [RoboFlow
+Blog](https://blog.roboflow.com/whats-new-in-yolov8/)
+"""
+
+"""
+First we will create a instance of backbone which will be used by our yolov8 detector
+class. 
+
+Backbones available in KerasCV:
+
+1.   Without Weights:
+
+    1.   yolo_v8_xs_backbone
+    2.   yolo_v8_s_backbone
+    3.   yolo_v8_m_backbone
+    4.   yolo_v8_l_backbone
+    5.   yolo_v8_xl_backbone
+
+2. With Pre-trained coco weight:
+
+    1.   yolo_v8_xs_backbone_coco
+    2.   yolo_v8_s_backbone_coco
+    2.   yolo_v8_m_backbone_coco
+    2.   yolo_v8_l_backbone_coco
+    2.   yolo_v8_xl_backbone_coco
+
+
+
+"""
+
+backbone = keras_cv.models.YOLOV8Backbone.from_preset(
+    "yolo_v8_s_backbone"  # We will use yolov8 small backbone
```

Comment:
Perhaps we should recommend using a pre-trained backbone by default? (e.g. yolo_v8_s_backbone_coco)

Reply:
I have already used `yolo_v8_s_backbone` in the notebook but didnt update the .py file here